uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsMockMvcTests.java

```diff
@@ -397,6 +398,109 @@ void create_user_without_email() throws Exception {
                                         .put("error", "invalid_scim_resource"))));
     }
 
+    @Test
```

Comment:
I can't figure out from the test code why the alias is supposed to be ignored in all the tests in this class. Maybe update the comments here to explain?

Reply:
fixed with commit 22f80aa